substrate/frame/contracts/proc-macro/src/lib.rs

```diff
@@ -497,9 +497,14 @@ fn expand_docs(def: &EnvDef) -> TokenStream2 {
 fn expand_env(def: &EnvDef, docs: bool) -> TokenStream2 {
 	let impls = expand_impls(def);
 	let docs = docs.then_some(expand_docs(def)).unwrap_or(TokenStream2::new());
+	let stable_count = def.host_funcs.iter().map(|f| if f.is_stable { 1 } else { 0 }).sum::<u16>();
```

Comment:
Interesting way to do it instead of `filter(|f| f.is_stable()).count()` :grin:

Reply:
ahah right that looks better